### PR TITLE
Change Creative Commons sentence to <p>.

### LINKED ISSUE #104: Creative Commons sentence should be a <p> instead of a <h4>

### PR DESCRIPTION
**Pull request by yangshun**
Fixes #104

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -173,9 +173,9 @@ <h4 class="section-title">Other stories &amp; resources</h4>
       <div class="container">
         <a href="https://www.mozilla.org/en-US" class="logo" id="#mozilla">mozilla</a>
         <div class="footer-content">
-          <h4>
-            Except where otherswise <a href=" https://www.mozilla.org/en-US/about/legal/#site">noted</a>, content of this site is licensed under the <a href="https://www.mozilla.org/en-US/foundation/licensing/website-content">Creative Commons</a> <a href="http://creativecommons.org/licenses/by-sa/3.0">Attribution Share-Alike License v3.0</a> or any later.
-          </h4>
+          <p class="footer-license">
+            Except where otherwise <a href=" https://www.mozilla.org/en-US/about/legal/#site">noted</a>, content of this site is licensed under the <a href="https://www.mozilla.org/en-US/foundation/licensing/website-content">Creative Commons</a> <a href="http://creativecommons.org/licenses/by-sa/3.0">Attribution Share-Alike License v3.0</a> or any later.
```
Comment (cvan) [inline public/index.html]: thanks for fixing the typo. could you remove the space after `href="` too?

Comment (yangshun) [inline public/index.html]: Done! (:

**Comment by yangshun** [comment]
On a side note, I can't seem to run the build step locally. How are the assets compiled and minified?

**Comment by bonfirered** [comment]
We used CodeKit on this and in hindsight that probably wasn't a good idea. I can help change it over to something more universal if that's helpful.

**Comment by yangshun** [comment]
@bonfirered it's ok, seems like it will take some time to do so. Is the PR ok to be merged? (:

**Comment by ryanschmidt** [comment]
@yangshun that previous reply from @bonfirered was me. Multiple accounts... oops. And yes, this PR looks good.

**Comment by cvan** [comment]
rebase needed

**Comment by yangshun** [comment]
@cvan Ok, rebased.

**Pull request merged**